Changelog.md

```diff
@@ -5,10 +5,12 @@ Language Features:
 
 Compiler Features:
  * Code Generator: Use SELFBALANCE for ``address(this).balance`` if using Istanbul EVM
+   sizes exceeding unsigned integer
```

Comment:
This also need to go into the bugfixes section ;)

Reply:
oh damn